metron-deployment/README.md

```diff
@@ -1,175 +1,127 @@
-- Storm's topology.classpath is not updated with the Metron service install and needs to be updated separately.
-- Several configuration parameters used when installing the Metron service could (and should) be grabbed from Ambari.  Install will require them to be manually entered.
-- Need to handle upgrading Metron
-
-## RPMs
-RPMs can be built to install the components in metron-platform. These RPMs are built in a Docker container and placed into `target`.
-
-Components in the RPMs:
-- metron-common
-- metron-data-management
-- metron-elasticsearch
-- metron-enrichment
-- metron-parsers
-- metron-pcap
-- metron-solr
-- stellar-common
-
-### Prerequisites
-- Docker.  The image detailed in: `metron-deployment/packaging/docker/rpm-docker/README.md` will automatically be built (or rebuilt if necessary).
-- Artifacts for metron-platform have been produced.  E.g. `mvn clean package -DskipTests` in `metron-platform`
-
-The artifacts are required because there is a dependency on modules not expressed via Maven (we grab the resulting assemblies, but don't need the jars).  These are
-- metron-common
-- metron-data-management
-- metron-elasticsearch
-- metron-enrichment
-- metron-indexing
-- metron-parsers
-- metron-pcap-backend
-- metron-solr
-- metron-profiler
-- metron-config
-
-### Building RPMs
+This will make the Metron available as an installable service within Ambari.
+
+#### Where can I get more information?
+
+The MPack can allow Metron to be installed and then kerberized, or installed on top of an already kerberized cluster.  This is done through Ambari's standard kerberization setup.  Using the MPack is preferred, but instructions for Kerberizing manually can be found at [Kerberos-manual-setup.md](Kerberos-manual-setup.md). These instructions are reference by the Ambari Kerberos install instructions and include commands for setting up a KDC.
+
+* [Metron Mpack](packaging/ambari/metron-mpack)
+
+## How do I build RPM packages?
+
+#### Who is this for?
+If you want to manually install Metron on an RPM-based system like CentOS or if you want a guided installation process using the Ambari Mpack on an RPM-based system, then this is the right option for you.
+
+#### What does this do?
+This builds installable RPM packages that allow you to install Metron on an RPM-based operating system like CentOS.
+
+#### How do I do it?
 ```
 cd metron-deployment
 mvn clean package -Pbuild-rpms
 ```
 
-The output RPM files will land in `target/RPMS/noarch`.  They can be installed with the standard
-```
-rpm -i <package>
-```
+The RPM packages will land in `metron-deployment/packaging/docker/rpm-docker/RPMS/noarch`.   
+
+#### Where can I get more information?
 
-## Kibana Dashboards
+* [RPM Docker](packaging/docker/rpm-docker)
 
-The dashboards installed by the Kibana custom action are managed by the dashboard.p file.  This file is created by exporting existing dashboards from a running Kibana instance.
+## How do I build DEB packages?
```

Comment:
Can we add a brief note in here that DEB support is in an alpha state right now?  Based on the discuss thread, I think that's sufficient to let users know that there may be issues, but that they're welcome to try.

I think once the full dev env + testing shake out some more, then we reevaluate the state we're in and adjust accordingly.